src/models/device.js

```diff
@@ -33,14 +34,15 @@ class Device {
             result[0].clientip,
             result[0].ios_version,
             result[0].ipa_version,
-            result[0].notes
+            result[0].notes,
+            result[0].enabled
         );
     }
-    static async create(uuid, config = null, lastSeen = null, clientip = null, iosVersion = null, ipaVersion = null, notes = null) {
+    static async create(uuid, config = null, lastSeen = null, clientip = null, iosVersion = null, ipaVersion = null, notes = null, enabled = null) {
```

Comment:
```suggestion
    static async create(uuid, config = null, lastSeen = null, clientip = null, iosVersion = null, ipaVersion = null, notes = null, enabled = true) {
```
I think devices should be enabled by default tbh